ui/app/(prowler)/resources/page.tsx

```diff
@@ -0,0 +1,216 @@
+import { Spacer } from "@nextui-org/react";
+import { format, parseISO } from "date-fns";
+import { Suspense } from "react";
+
+import { getResourceFields, getResources } from "@/actions/resources";
+import { getScans } from "@/actions/scans";
+import { FilterControls } from "@/components/filters";
+import { SkeletonTableResources } from "@/components/resources/skeleton/skeleton-table-resources";
+import { ColumnResources } from "@/components/resources/table/column-resources";
+import { ContentLayout } from "@/components/ui";
+import { DataTable, DataTableFilterCustom } from "@/components/ui/table";
+import {
+  createDict,
+  extractFiltersAndQuery,
+  extractSortAndKey,
+  hasDateOrScanFilter,
+  replaceFilterFieldKey,
+} from "@/lib";
+import { ResourceProps, SearchParamsProps } from "@/types";
+
+export default async function Resources({
+  searchParams,
+}: {
+  searchParams: SearchParamsProps;
+}) {
+  const searchParamsKey = JSON.stringify(searchParams || {});
+
+  // Check if the searchParams contain any date or filter
+  const hasDateOrScan = hasDateOrScanFilter(searchParams);
+
+  const { filters } = extractFiltersAndQuery(searchParams);
+  filters["page[size]"] = "100"; // TODO: Remove page[size] 100 when metadata endpoint implemented
+
+  if (!hasDateOrScan) {
+    const scansData = await getScans({
+      filters: {
+        "fields[scans]": "inserted_at",
+      },
+    });
+
+    if (scansData?.data?.length !== 0) {
+      const latestScandate = scansData.data?.[0]?.attributes?.inserted_at;
+      const formattedDate = format(parseISO(latestScandate), "yyyy-MM-dd");
```

Comment:
We have to handle this error 

![image](https://github.com/user-attachments/assets/179684cc-5ba2-4669-8391-7ec6c0a582dc)